docs/api_requests/block-io-engine.md

```diff
@@ -0,0 +1,66 @@
+# Block device IO engine
+
+Prior to Firecracker 1.0.0, the emulated block device uses a synchronous IO
```

Comment:
```suggestion
Prior to Firecracker 1.0.0, the emulated block device used a synchronous IO
```

Reply:
I get your reasoning, but it just sounds a bit weird to me to keep present tense here. How about rephrasing to something like:

```suggestion
For all Firecracker versions prior to v1.0.0, the emulated block device uses a synchronous IO 
engine for executing the device requests, based on blocking system calls for.
```
so the it would be clear that firecracker supports Sync for all versions and the next paragraph specifies that Async is only available starting with Firecracker v1.0.0.

Reply:
done 👍🏻